fsr/fsr.ino

```diff
@@ -1,10 +1,14 @@
+#include "FastADC.h"
 #include <inttypes.h>
 
 #if !defined(__AVR_ATmega32U4__) && !defined(__AVR_ATmega328P__) && \
     !defined(__AVR_ATmega1280__) && !defined(__AVR_ATmega2560__)
   #define CAN_AVERAGE
+  #define FASTADC 1
```

Comment:
This if statement is for boards that ARE teensy. Also we should be able to just re-use the CAN_AVERAGE thing instead of defining a new FASTADC macro

Reply:
Yup, sorry i didnt do it right ;(